view/simple/src/test/kotlin/io/holunda/polyflow/view/simple/service/ConvertersKtTest.kt

```diff
@@ -0,0 +1,39 @@
+package io.holunda.polyflow.view.simple.service
+
+import io.holunda.camunda.taskpool.api.business.DataEntryAnonymizedEvent
+import io.holunda.camunda.taskpool.api.business.Modification
+import io.holunda.polyflow.view.DataEntry
+import io.holunda.polyflow.view.addModification
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.OffsetDateTime
+
+class ConvertersKtTest {
+    @Test
+    fun `should apply anonymization`() {
+        val now = OffsetDateTime.now()
+        val event = DataEntryAnonymizedEvent(
+            "type", "id", "ANONYMIZED", listOf("SYSTEM"), Modification(now, "SYSTEM", "anonymize", "anonymize")
```

Comment:
Please use the Modification created by the user, not system.

Reply:
I mean the modification is part of the Event / Command. Either you want it to say User_XYZ triggered the anonymization or you just pass SYSTEM or ANOYNOMOUS in the first place. After all it is the same place where you specify the `anonymizedUsername`